### ISSUE TITLE
2024 Osmo compatible device for kindergartener

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#1016]
In #967 I found the Fire HD devices we current  have are no longer compatible with Osmo so newer installation is possible, then I want to get a new device that is compatible https://github.com/kinu-garage/30y-130s_life/issues/1695#issuecomment-1872266908

**Comment by 130s** [comment]
TLDR;
- Despite the parent company of the vendor of Osmo (ref. [wikipedia](https://en.wikipedia.org/wiki/Osmo_(game_system))) filing bankruptcy, which makes the future of Osmo very merky as of today, I think Osmo is a brilliant idea and AFAIK the apps work without cloud access so even in case of closure of the vendor we can still use them.
- I found a compatible device that is also powerful enough for my ideal goal of device, I can go with [Fire HD 10 Plus (amazon.com, certified refurbished *1)](https://www.amazon.com/gp/product/B08F5M223W/ref=ewc_pr_img_2?smid=ATVPDKIKX0DER&psc=1) 4GB RAM (even for 5-yr old child, low performance due to the short of RAM can happen and negatively affect the experience, so I want higher spec as much as possible. FYI for the device we currently have, spec: https://github.com/kinu-garage/hut_10sqft/issues/843#issuecomment-1586343820 [1.5GB RAM](https://www.amazon.com/All-New-Amazon-Fire-HD-8-Tablet-With-Alexa/dp/B01J94SWWU)).
- Q. What's the diff of Amazon Fire HD 10 and 10 plus? 
- A. [Amazon Fire HD 10 (2021) vs. Fire HD 10 Plus (2021): Should you grab the plus? (androidpolice.com)](https://www.androidpolice.com/amazon-fire-hd-10-2021-vs-fire-hd-10-plus-2021/#:~:text=For%20most%20people%2C%20there's%20no,to%20life%20a%20little%20more.)
   > For most people, there's no reason not to get the Amazon Fire HD 10 Plus over the standard model. While both tablets are basically identical, the extra GB of RAM on the Plus model does make the tablet come to life a little more.
- Q. Is there a plan for Osmo to be released on Android?
- A. [None (playosmo.zendesk.com)](https://playosmo.zendesk.com/hc/en-us/articles/235182707-When-do-you-plan-to-launch-an-Android-version#:~:text=Please%20see%20this%20list%20of,Osmo%20into%20other%20Android%20tablets.) (viewed on 20240411).
- Q. I see [Amazon  Fire HD 10  2023 Release (bestbuy.com)](https://www.bestbuy.com/site/amazon-fire-hd-10-10-1-tablet-2023-release-32gb-black/6562284.p?skuId=6562284) and [2021 release (amazon.com)](https://www.amazon.com/dp/B08BX8CW9V/ref=pav_fromAsin_B08F6FYN6B_toAsin_B08BX7FV5L?th=1). What's diff?
- A. Performance boost according to [CNN](https://www.cnn.com/cnn-underscored/reviews/amazon-fire-hd-10-2023)
   > The Fire HD 10 (2023) brings a slight performance boost and an extra hour of battery life over the 2021 model, along with a better front-facing camera for video calls, in a slightly lighter housing.


*1...I have one good experience with a certified refurbished item sold on amazon.com https://github.com/kinu-garage/hut_10sqft/issues/885, so I can try again.

**Comment by 130s** [comment]
Kids started playing [Osmo Detective Agency](https://www.amazon.com/Osmo-Detective-Agency-Explores-Required/dp/B07GJ1XSGP?th=1). Some finding in the first 20 mins:
- [Reflector Adapter](https://www.amazon.com/Osmo-Reflector-Generation-Horizontal-Positioning/dp/B08HRT1JRX) seems required, which must have come with the original Osmo reflector but we can't find at home.
   - For now we're trying to work it around by keeping the reflector from going all the way down close to the Fire device, and keeping a small space in between.
- App has got stuck twice already, rebooting the OS got it working again. Hmmm..
